e2e/test-component/scenarios/embedding-sdk/styles-tests.cy.spec.tsx

```diff
@@ -56,7 +55,7 @@ describe("scenarios > embedding-sdk > styles", () => {
           authConfig={DEFAULT_SDK_AUTH_PROVIDER_CONFIG}
           theme={theme}
         >
-          <CreateQuestion />
+          <InteractiveQuestion isSaveEnabled />
```

Comment:
I think `isSaveEnabled` already defaults to `true` would we still need this?
https://github.com/metabase/metabase/blob/fd4b569fa2e2a357a82451c952c86c5f4ca48d55/enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/InteractiveQuestionProvider.tsx#L54

It portrays the wrong narative that this prop is required.

Reply:
I always forget that `isSaveEnabled` already defaults to `true`, fixed in 90d3517b2adbb74922ad15b0fb58a9e89e419f8c